examples/seq2seq/run_seq2seq.py

```diff
@@ -523,7 +551,7 @@ def compute_metrics(eval_preds):
     if training_args.do_eval:
         logger.info("*** Evaluate ***")
 
-        results = trainer.evaluate()
+        results = trainer.evaluate(max_length=data_args.val_max_target_length, num_beams=data_args.eval_beams)
```

Comment:
`eval_beams` was never used before this PR, we should pass it to `evaluate` and `predict`.